spec/models/iiif_record_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe IIIFRecord, :type => :model do
+  let(:solr_doc) {{
+    "title_citation_display" => ['This is the Title'],
+    "summary_note_display"   => ['This is a note about it.'],
+    "pub_date_display"       => ['1970'],
+    "language_facet"         => ['English', 'Spanish'],
+    "language_code_s"        => ['eng'],
+    "author_roles_1display"  => ['{"secondary_authors":[],"translators":["Translator, Bob"],"editors":[],"compilers":[],"primary_author":"Author, Alice"}']
+  }}
+  subject { described_class.new solr_doc }
+  it 'produces iiif json' do
+    iiif_json = {
+      label: 'This is the Title',
+      description: 'This is a note about it.',
+      metadata: [
+        { label: 'date', value: '1970' },
+        { label: 'language', value: ['English', 'Spanish'] },
```

Comment:
Some clarification would be good.  I see two permutations in the spec:

The first is a single simple value:

```
{ "label": "Author", "value": "Anne Author" }
```

The second is multiple complex values:

```
      { "label": "Published", "value": [
        {"@value": "Paris, circa 1400", "@language": "en"},
        {"@value": "Paris, environ 1400", "@language": "fr"}
      ]
    }
```

There's no example for how to handle multiple simple values (nor a single complex value, for that matter).